pkg/system/reconciler.go

```diff
@@ -179,11 +186,14 @@ func NewReconciler(
 	r.SecretOp.Name = r.Request.Name + "-operator"
 	r.SecretAdmin.Name = r.Request.Name + "-admin"
 	r.SecretEndpoints.Name = r.Request.Name + "-endpoints"
-	r.AzureContainerCreds.Name = r.Request.Name + "-azure-container-creds"
 	r.AWSCloudCreds.Name = r.Request.Name + "-aws-cloud-creds"
 	r.AWSCloudCreds.Spec.SecretRef.Name = r.Request.Name + "-aws-cloud-creds-secret"
+	r.AzureContainerCreds.Name = r.Request.Name + "-azure-container-creds"
 	r.AzureCloudCreds.Name = r.Request.Name + "-azure-cloud-creds"
 	r.AzureCloudCreds.Spec.SecretRef.Name = r.Request.Name + "-azure-cloud-creds-secret"
+	r.GCPBucketCreds.Name = r.Request.Name + "-gcp-container-creds"
```

Comment:
Please change the "-gcp-container-creds" to "-gcp-bucket-creds"

Reply:
Done, please check.